Modelica/Blocks/Logical.mo

```diff
@@ -945,14 +945,16 @@ The output <code>Q</code> is set by the input <code>S</code>, is reset by the in
       Documentation(
         info="<html>
 <p>
-This block is used by connecting a&nbsp;(possibly) <strong>time varying</strong>
-input condition for the Boolean input variable&nbsp;<strong>u</strong>.
-If this input variable is <strong>false</strong>, the simulation will
+This block asserts that the Boolean input&nbsp;<strong>u</strong> is <strong>true</strong>.
+If&nbsp;<strong>u</strong> is <strong>false</strong>, the simulation will
 either abort with an error (if <strong>assertionLevel</strong> is set to
 <strong>error</strong>), or issue a&nbsp;warning and continue (if
 <strong>assertionLevel</strong> is set to <strong>warning</strong>).
-The <strong>message</strong> will be included in the message text of
-the error or warning.
+The <strong>message</strong> will be reported in either case.
+Typically, this block is used by connecting a&nbsp;(possibly) <strong>time
+varying</strong> outside condition and the input&nbsp;<strong>u</strong>
+to terminate the simulation if the condition fails.
```

Comment:
As there is another block for _terminating_ the simulation, shouldn't we avoid using the term _terminate_ in the current context?

Reply:
"stop" could work. Short and without the additional meaning.

Reply:
I agree.

Reply:
Or "abort", which has the additional meaning of not terminating successfully.

Reply:
Switched to stop.